search/releases/5.6/index.adoc

```diff
@@ -1,3 +1,69 @@
 :awestruct-layout: project-releases-series
 :awestruct-project: search
-:awestruct-series_version: "5.6"
\ No newline at end of file
+:awestruct-series_version: "5.6"
+
+=== Experimental Elasticsearch integration
+
+A new, experimental Elasticsearch integration was introduced in this series.
+
+With the Elasticsearch integration, rather than indexing your entities directly by managing the Lucene resources,
+Hibernate Search will be sending RPCs to an Elasticsearch cluster to achieve a very similar purpose:
+after all it is also Lucene based, so the feature match is extremely close!
+
+This means that Hibernate Search will able to transparently map all the current features
+to this new alternative backend,
+and by so doing give you more architectural choices at minimum required changes in your applications:
```

Comment:
This sentence sounds weird. Maybe it's just too long.

Reply:
Haha, this time it's not me :) That's copy/pasted text from an old blog post.

WDYT of "Since Hibernate Search will transparently map all the current features to this new alternative backend, you will be able to switch to a new architecture with only minimal changes in your applications."
And keep the following sentence, "For most users the differences will be mostly in configuration details"

Reply:
Yes, it's better.